torchgeo/datasets/geo.py

```diff
@@ -725,13 +719,29 @@ def __getitem__(self, query: BoundingBox) -> dict[str, Any]:
             # with the default fill value and dtype used by rasterize
             masks = np.zeros((round(height), round(width)), dtype=np.uint8)
 
-        sample = {"mask": torch.tensor(masks), "crs": self.crs, "bbox": query}
+        # Use array_to_tensor since rasterize may return uint16/uint32 arrays.
+        masks = array_to_tensor(masks)
+
+        sample = {"mask": masks, "crs": self.crs, "bbox": query}
 
         if self.transforms is not None:
             sample = self.transforms(sample)
 
         return sample
 
+    def _get_label(self, feature: Any) -> int:
```

Comment:
Let's make this function public so it shows up in the docs. Also, shouldn't this be:
```suggestion
    def _get_label(self, feature: Feature) -> int:
```

Reply:
fiona.Feature doesn't exist in the min req version. I tried Union[Feature, dict[str, Any]] but it fails because the fiona.Feature doesn't exist. I tried to define Feature in a variable way (import fiona.Feature, but catch ImportError and define a placeholder) but it also doesn't work because the different definition of Feature would be caught as a re-definition.

Reply:
In the old fiona version, features are represented as dicts.

Reply:
It looks like `fiona.Feature` was introduced in fiona 1.9, which is about a year old now. I'm okay with bumping the minimum supported version of fiona to 1.9 if that makes this easier to annotate. You would modify `pyproject.toml` and `requirements/min-reqs.old`.

Alternatively, you can do `"fiona.Feature"` to prevent evaluation during run-time.

Reply:
To fix the RtD tests, you can add `fiona.Feature` to https://github.com/microsoft/torchgeo/blob/v0.5.1/docs/conf.py#L59 since it isn't actually documented.

Reply:
Ah ok perfect, I used `"fiona.Feature"`.